src/main/scala/mesosphere/marathon/api/v2/ModelValidation.scala

```diff
@@ -21,8 +21,18 @@ object ModelValidation {
   def checkGroup(
     group: Group,
     path: String,
-    parent: PathId): Iterable[ConstraintViolation[V2Group]] =
-    checkGroup(V2Group(group), path, parent)
+    parent: PathId,
+    maximum: Int): Iterable[ConstraintViolation[V2Group]] = {
```

Comment:
Sorry for not noting that before. I'd like some like `maxApps` better than `maximum`